db/seeds.rb

```diff
@@ -1,7 +1,14 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'factory_girl_rails'
+require 'spec_helper'
+require 'rails_helper'
+require 'rspec'
+require 'rubygems'
```

Comment:
@dfaulken as it would turn out I had gotten a bit carried away when trying to find out why those methods were not being imported properly -- none of those are necesary